### PR TITLE
Better error message for NamedConstructorAggregateFactory assertion

### LINKED ISSUE #174: Missing message from assertion in NamedConstructorAggregateFactory
https://github.com/qandidate-labs/broadway/blob/26ab0d8e527c990169727a92d59faf24f0f256cd/src/Broadway/EventSourcing/AggregateFactory/NamedConstructorAggregateFactory.php#L30

### PR DESCRIPTION
**Pull request by wjzijderveld**
Fixes #174

### CODE REVIEW

**Comment by mbadolato** [comment]
:+1:

File: src/Broadway/EventSourcing/AggregateFactory/NamedConstructorAggregateFactory.php [hunk 1]
```diff
@@ -27,9 +27,13 @@ public function __construct($staticConstructorMethod = 'instantiateForReconstitu
      */
     public function create($aggregateClass, DomainEventStreamInterface $domainEventStream)
     {
-        Assert::true(method_exists($aggregateClass, $this->staticConstructorMethod));
-
         $methodCall = sprintf('%s::%s', $aggregateClass, $this->staticConstructorMethod);
+
+        Assert::true(
+            method_exists($aggregateClass, $this->staticConstructorMethod),
+            sprintf('NamedConstructorAggregateFactory expected %s to exists', $methodCall)
```
Comment (othillo) [inline src/Broadway/EventSourcing/AggregateFactory/NamedConstructorAggregateFactory.php]: to exist

Comment (wjzijderveld) [inline src/Broadway/EventSourcing/AggregateFactory/NamedConstructorAggregateFactory.php]: Good one, that's what you get when you rearrange your sentence last minute :)

**Comment by simensen** [comment]
:+1: I've run into this multiple times and was fortunate enough to have @wjzijderveld there this time to keep track of these things and submit issues **and** fixes for broadway today. :)

**Comment by othillo** [comment]
:+1:

**Pull request merged**